doc/Command-Reference.md

```diff
@@ -2379,6 +2379,34 @@ VLAN interface names take the form of `vlan<vlan_id>`. E.g., VLAN 100 will be na
   admin@sonic:~$ sudo config interface vlan100 ip remove 10.11.12.13/24
   ```
 
+**config interface pfc priority <interface_name> <priority> (on | off) (Versions >= 201904)**
```

Comment:
Remove `(Versions >= 201904)`, as this command is not in that version.

Reply:
@jleveque Fixed